actionview/RUNNING_UNIT_TESTS.rdoc

```diff
@@ -2,13 +2,13 @@
 
 The easiest way to run the unit tests is through Rake. The default task runs
 the entire test suite for all classes. For more information, checkout the
-full array of rake tasks with "rake -T"
+full array of rake tasks with +rake -T+.
```

Comment:
This needs to use `<tt>`, not `+`, since it's more than one word: `<tt>rake -T</tt>`